### ISSUE TITLE
🐞 Trigger doesn't work even when the requested access in accessibility menu for Loop is enabled

### ISSUE THREAD

**Issue by Sherifoz** [MrKai77/Loop#496]
### Bug Description

Hi,

Thanks a lot for working on this great program! I used the program for several months without any issues but now it stopped working appropriately. I can still open the program and change the options but no effect happens, so I'm unable to use it. I have the trigger button as left Ctrl, and when I double click on it, the radial menu doesn't appear and nothing happens. Please advise, thanks!

### Steps To Reproduce

Hi,

Thanks a lot for working on this great program! I used the program for several months without any issues but now it stopped working appropriately. I can still open the program and change the options but no effect happens, so I'm unable to use it. I have the trigger button as left Ctrl, and when I double click on it, the radial menu doesn't appear and nothing happens. Please advise, thanks!

### Expected Behavior

Hi,

Thanks a lot for working on this great program! I used the program for several months without any issues but now it stopped working appropriately. I can still open the program and change the options but no effect happens, so I'm unable to use it. I have the trigger button as left Ctrl, and when I double click on it, the radial menu doesn't appear and nothing happens. Please advise, thanks!

### Actual Behavior

Hi,

Thanks a lot for working on this great program! I used the program for several months without any issues but now it stopped working appropriately. I can still open the program and change the options but no effect happens, so I'm unable to use it. I have the trigger button as left Ctrl, and when I double click on it, the radial menu doesn't appear and nothing happens. Please advise, thanks!

### MacOS Version

Sonoma 14.5

### Loop Version

1.0.0 (1030)

### Final Checks

- [X] My issue is written in English
- [X] My issue title is descriptive
- [X] This is a single bug (multiple bugs should be reported individually)

**Comment by Sherifoz** [comment]
And forgive me if my entry is not ideal. That's my first time to write anything on Github as a normal user, because I'm a fan of Loop! :) Please let me know if you need any extra info

**Comment by JaceThings** [comment]
Please try this

https://github.com/MrKai77/Loop/issues/418#issuecomment-2178849660

**Comment by Sherifoz** [comment]
> Please try this
> 
> [#418 (comment)](https://github.com/MrKai77/Loop/issues/418#issuecomment-2178849660)

That did the trick and solved it. Thanks a lot!
